apps/web/src/services/iam/components/UserGroupHeader.vue

```diff
@@ -0,0 +1,59 @@
+<script setup lang="ts">
+import { PHeading, PButton } from '@cloudforet/mirinae';
+
+import { i18n } from '@/translations';
+
+import { USER_GROUP_MODAL_TYPE } from '@/services/iam/constants/user-group-constant';
+import { useUserGroupPageStore } from '@/services/iam/store/user-group-page-store';
+
+const userGroupPageStore = useUserGroupPageStore();
+const userGroupPageState = userGroupPageStore.state;
+
+const handleCreateUserGroup = (type: string) => {
+    switch (type) {
+    case USER_GROUP_MODAL_TYPE.ADD: userGroupPageStore.updateModalSettings({
+        type,
+        title: i18n.t('IAM.USERGROUP.MODAL.ADD.TITLE'),
+        visible: USER_GROUP_MODAL_TYPE.ADD,
+        themeColor: 'primary',
+    }); break;
+    default:
+        break;
+    }
+};
```

Comment:
question: There's only a create button here, is there a specific reason for using a switch-case statement?

Reply:
I wanted to provide consistency and scalability with other components.